### DISCUSSION TITLE
lurker container specs

### DISCUSSION THREAD

**Discussion by cdm-anv** [secureCodeBox/secureCodeBox#2559] [Q&A]
Dear Community, 

Securecodebox injects the lurker container into each scan job/pod. We need to customize certain specifications of the lurker container, such as securityContext or automountServiceToken properties. However, I haven't found a solution to achieve this. We are using the latest Helm charts and Kustomize to install Securecodebox on our Kubernetes cluster.

Is there a way to accomplish this customization?

**Comment by J12934** [comment]
Hi @cdm-anv 
No that's not supported right now.
The goal was to have the lurker run with as little rights as required so that it works with almost all rules/policies.
The securityContext set on the lurker should be enough to have it run in the strict pod security standard.

The automountServiceToken token is explicitly set to true as the lurker needs that token to talk to the kubernetes api to find out weather the scanner pod is finished.

Can you go into more details what your requirements reguarding security content are that the lurker doesn't meet yet?
Would be interessting to see if there is something we could set in addition to the current settings.
